packages/databricks-vscode/src/configuration/auth/AuthProvider.ts

```diff
@@ -125,6 +125,14 @@ export abstract class AuthProvider {
 }
 
 export class ProfileAuthProvider extends AuthProvider {
+    static async from(profile: string, checked = false) {
+        const host = await new WorkspaceClient(
+            ProfileAuthProvider._getSdkConfig(profile)
+        ).apiClient.config.getHost();
```

Comment:
`workspaceClient.apiClient.config` is the config we pass to the workspace client in the first place (created by `_getSdkConfig`). Is there a hidden reason behind creating workspace client? Can we document it here then?

Reply:
Good catch. No reason. Removed.